### PR TITLE
appendTo feature for reveal

### PR DESCRIPTION
**Pull request by phifa**

### CODE REVIEW

File: js/foundation.reveal.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ class Reveal {
   _makeOverlay(id) {
     var $overlay = $('<div></div>')
                     .addClass('reveal-overlay')
-                    .appendTo('body');
+                    .appendTo('this.options.appendTo');
```
Comment (Owlbertz) [inline js/foundation.reveal.js]: In order to have the code aligned I would like to see ` .appendTo($(this.options.appendTo));

**Review by Owlbertz — APPROVED** [review]
Seems like a useful feature to me.
Just one small think I commented on a line.
I would like to have feedback from another @zurb/yetinauts on that since it changes the public API.

File: js/foundation.reveal.js (hunk 1 above)
Comment (kball) [inline js/foundation.reveal.js]: I agree, putting it in the quotes won't work, because jquery will try to find that string, rather than the variable.

Comment (kball) [inline js/foundation.reveal.js]: In fact, I think I need to change this to a review because without this change this PR is not correct.

**Review by kball — CHANGES_REQUESTED** [review]
We need to change the overlay append to properly reference the appendTo variable rather than a string.

**Comment by kball** [comment]
I'm going to just do the update for the bug in place and get this merged

**Pull request merged**